subsys/dm/Kconfig

```diff
@@ -4,7 +4,13 @@
 # SPDX-License-Identifier: LicenseRef-Nordic-5-Clause
 #
 
-if NRF_DM
+config NRF_DM_SM
+	bool "nRF DM (Distance Measurement) submodule [EXPERIMENTAL]"
```

Comment:
in the documentation we talk about "The Distance Measurement module", maybe we should align the naming here.